src/app/server/Server.h

```diff
@@ -386,13 +386,15 @@ class Server
         return System::SystemClock().GetMonotonicMicroseconds64() - mInitTimestamp;
     }
 
-    static Server & GetInstance() { return sServer; }
+    static Server & GetInstance()
+    {
+        static Server * sServer = new Server();
```

Comment:
This is going to get detected as a leak by leak-detection tools, no?

Reply:
It's not a real leak, since the heap memory will still be reclaimed by the OS when the program exits (unless the Server destructor has some important side effects that will cause real leaks if they're not run). Our ASAN runs detect the intialization-order-fiasco but don't flag this.

Reply:
Ideally I would not have the static instance at all, and create a single that each platform must construct and destroy. But that could be a breaking change unless we have a place in common code to put the Server member.

Reply:
Yeah, I'm not saying it's a real leak, just that it will lead to false positives in tools....

Reply:
I think that most tools (most that I know) will report that as not freed but still reachable memory because it's a static variable and such report in most cases is not fatal. There is a "ton" of allocations like that in glib library :D